app/services/exports/trainee_search_data.rb

```diff
@@ -269,6 +269,13 @@ def trainee_education_phase(trainee)
       trainee.course_education_phase&.upcase_first
     end
 
+    def start_year(trainee)
```

Comment:
I don't this logic should be in here. Might be better on the trainee for now. Trainee also has `academic_cycle` which  already gives you this.